internal/k8s/kubernetes.go

```diff
@@ -49,12 +39,6 @@ func GetPodNameByLabel(podsClient coreV1Types.PodInterface, label string) string
 	return gitlabToolboxPodName
 }
 
-func DeletePodByName(podsClient coreV1Types.PodInterface, podName string) {
-	err := podsClient.Delete(context.TODO(), podName, metaV1.DeleteOptions{})
-	if err != nil {
-		log.Println(err)
-	}
-}
```

Comment:
Why removing it?

Reply:
It is a nice utility that I see value downstream, if we stoped using today.

Reply:
the function is not in use, there is no other part of the code that needs it

Reply:
Approved with reservations, prefer we hold the deletion of artifacts.